### ISSUE TITLE
cross_entropy_error.hpp not found in master branch

### ISSUE THREAD

**Issue by nitolpalak** [mlpack/mlpack#3165]
I have used mlpack 3.4.2 for my API, but for some complications I needed to build mlpack from master branch and now when I am trying to build my API it is showing that `"mlpack/methods/ann/loss_functions/cross_entropy_error.hpp" file not found`. After checking the mlpack/methods/ann/loss_functions/ I see that there is no cross_entropy_error.hpp file in the source from master branch but the file exists in 3.4.2 release. **Was the cross_entropy_error completely removed from the master branch? Or was it's name changed into something else**? Thanks in advance.

**Comment by hello-fri-end** [comment]
Hi @nitolpalak! `Cross Entropy Error` in mlpack is an [alias](https://github.com/mlpack/mlpack/blob/master/src/mlpack/methods/ann/loss_functions/binary_cross_entropy_loss.hpp#L109) for `Binary Cross Entropy Loss`.
